src/appMain/androidSmartDeviceLink.ini

```diff
@@ -407,3 +407,13 @@ RpcPassThroughTimeout = 10000
 [RCModuleConsent]
 ; The period (in days) after which consent for module_id should be removed.
 PeriodForConsentExpiration = 30
+
+[ANDROID]
+BufferSize = 131072
+ReaderSocketAdress = "./localBleReader"
+ControlSocketAdress = "./localBleControl"
+WriterSocketAdress = "./localBleWriter"
+PrefferredMtu = 512
+SdlTesterServiceUUID = "00001101-0000-1000-8000-00805f9b34fb"
+MobileNotificationCharacteristic = "00001102-0000-1000-8000-00805f9b34fb"
+MobileResponceCharacteristic = "00001104-0000-1000-8000-00805f9b34fb"
```

Comment:
@VadymLuchko while we remember what it was, could you please add the description comment for each param and where it is used?

Reply:
fixed here 5b2d41c5b50c2acdd4877a2e14d10c056afab645